app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,132 @@
+class TweetsController < ApplicationController
+  before_action :set_tweet, only: %i[ show edit update destroy ]
+
+  # GET /tweets or /tweets.json
+  def index
+    @tweet = Tweet.new
+    @tweets = Tweet.all.order(created_at: :asc)
+  end
+
+  # GET /tweets/1 or /tweets/1.json
+  def show
+    @tweet
+    respond_to do |format|
+        format.html {render :show}
+        format.json { render :show, status: :ok, location: @tweet }
+    end
+  end
+
+
+  # GET /tweets/new
+  #def new
+  #  @tweet = Tweet.new
+  #end
+
+  # GET /tweets/1/edit
+  #def edit
+  #end
+
+
+  # POST /tweets or /tweets.json
+  def create
+    @tweet = Tweet.new(tweet_params)
+    if @tweet.save
+      flash[:success] = "New tweet successfully created!"
+      format.html { render :new}
+      format.json { render :show, status: :created, location: @tweet }
```

Comment:
You need to respond using the json schema you defined in your specs otherwise your test are going to fail